documentation/upgrade.md

```diff
@@ -1,3 +1,8 @@
+# Changes in version 4.0.0
+
+* Operator-sdk updated to v1.3.0
+* Installs Grafana 7.1.1 by default
+
```

Comment:
With the current settings the operator is installed in the namespace: grafana-operator-system
https://github.com/grafana-operator/grafana-operator/blob/4d90d97b530d7626daf6eb02b9baeaf332c25e59/config/default/kustomization.yaml
We can of course change this or It might be worht pointing out. What do you think?